llvm/include/llvm/TargetParser/Triple.h

```diff
@@ -831,6 +831,13 @@ class Triple {
            getArch() == Triple::spirv;
   }
 
+  // Tests whether the target is SPIR-V or SPIR. Currently, we use spir-based
+  // target triples to represent JIT compilation targets for SYCL. We will
+  // transition to using spir-v based target triples to represent JIT
+  // compilation targets for SYCL very soon. This helper function is used
+  // (instead of isSPIR) to ease that transition.
```

Comment:
Comments that attempt to predict the future have a tendency to either be wrong or to predict the past ;)